themes/classic/templates/_partials/microdata/head-jsonld.tpl

```diff
@@ -0,0 +1,87 @@
+{**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ *}
+<script type="application/ld+json">
+  {
+    "@context": "https://schema.org",
+    "@type": "Organization",
+    "name" : "{$shop.name}",
+    "url" : "{$urls.pages.index}",
+    "logo": {
+      "@type": "ImageObject",
+      "url":"{$urls.shop_domain_url}{$shop.logo}"
+    }
+  }
+</script>
+
+<script type="application/ld+json">
+  {
+    "@context": "https://schema.org",
+    "@type": "WebPage",
+    "isPartOf": {
+      "@type": "WebSite",
+      "url":  "{$urls.pages.index}",
+      "name": "{$shop.name}"
+    },
+    "name": "{$page.meta.title}",
+    "url":  "{$urls.current_url}"
+  }
+</script>
+
+{if $page.page_name == 'index'}
+  <script type="application/ld+json">
+    {
+      "@context": "https://schema.org",
+      "@type": "WebSite",
+      "url" : "{$urls.pages.index}",
+      "image": {
+        "@type": "ImageObject",
+        "url":"{$urls.shop_domain_url}{$shop.logo}"
```

Comment:
hello, this is bad
{$urls.shop_domain_url}{$shop.logo} --> {$shop.logo}

Reply:
@mgielecinski already modified in d40c7e2a905d28cbad0ecc85ed4b4d2531c45361

Reply:
in organization yes, but in line 60 still is error

Reply:
@mgielecinski oh yeah I see :) Thx a lot. @Progi1984 for info I opened #24233 (pull request) to correct this based on mgielecinski laser eyes recommendation.